docs/ReleaseInstructions.md

```diff
@@ -1,21 +1,53 @@
-### Release process
+# Release process
+
+### Prior to release: 1 week out
+
+* Ensure the version specified in `docs/config.ini`, `VERSION`, and `docs/badges.rst` align with the intended release version.
+* Make final call for which in-progress PRs are release critical.
+* Communicate with in-progress PR owners that code freeze is in effect, no PRs will be merged that are not release critical.
+* Wait 1 day after code-freeze for pre-release to be published.
+* Ensure latest pre-release is working via `pip install --pre autogluon` and testing to get an idea of how the actual release will function (Ideally with fresh venv). DO NOT RELEASE if the pre-release does not work.
+* Ensure each sub-module is working IN ISOLATION via `pip install --pre autogluon.{submodule}`.
+  * Ensure a fresh venv is used for each submodule.
+  * Doing this will avoid issues like in v0.4 release with `autogluon.text` crashing when installed standalone due to missing setup.py dependencies
+    * https://github.com/awslabs/autogluon/issues/1607
```

Comment:
We probably want to have a script to do this

Reply:
Yes, although we may only want to have this run in an on-demand fashion rather than automatically, as it would be expensive.